index.js

```diff
@@ -173,7 +173,24 @@ const create = (win, options) => {
 
 		const shouldShowInspectElement = typeof options.showInspectElement === 'boolean' ? options.showInspectElement : isDev;
 
+		const dictSuggestions = [];
+		for (let index = 0; index < props.dictionarySuggestions.length; index++) {
+			dictSuggestions.push(
+				{
+					id: 'dictSuggestions',
+					label: props.dictionarySuggestions[index],
+					visible: props.isEditable && hasText && props.misspelledWord,
```

Comment:
The `props.isEditable && hasText` check should be before the loop. We should not even do any work if it cannot be used.